plugins/modules/cobbler_system.py

```diff
@@ -280,8 +280,8 @@ def main():
 
         if system:
             # Update existing entry
-            system_id = None
-            if LooseVersion(str(conn.version())) >= LooseVersion('3.4.0'):
+            system_id = ''
+            if StrictVersion(str(conn.version())) >= StrictVersion('3.4.0'):
```

Comment:
Instead of using StrictVersion, why not simply use
```suggestion
            if LooseVersion(str(conn.version())) >= LooseVersion('3.4'):
```
?

Reply:
`StrictVersion` is for comparing versions of Python projects (on PyPI). `LooseVersion` is for comparing general version numbers. So I think `LooseVersion` is the right choice here.

Reply:
ok I will apply your suggestion. Here's another edge case to note in which `LooseVersion` will not work correctly, it's for pre-release versions as stated [here](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/compat/version.py#L238). But I think it's not used a lot so it should be fine.

Reply:
Pre-releases are part of a specific versioning scheme. If there is no concrete versioning scheme, like in this case, then using a more specialized class that's for another unrelated versioning scheme is a bad idea.